test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatAddPkgTests.cs

```diff
@@ -745,6 +746,123 @@ public async Task AddPkg_ConditionalAddWithUserInputFramework_Success(string pac
             }
         }
 
+        [Fact]
+        public async Task AddPkg_WithAdditionalSourceFeed_NoVersionSpecified_Success()
+        {
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                var projectFrameworks = "net472";
+                var packageFrameworks = "net472; netcoreapp2.0";
+                var projectA = XPlatTestUtils.CreateProject(ProjectName, pathContext, projectFrameworks);
+                var packageX = XPlatTestUtils.CreatePackage(frameworkString: packageFrameworks);
+                var packageXPath = Path.Combine(pathContext.WorkingDirectory, "packageX");
+                var mySourcePath = Path.Combine(pathContext.WorkingDirectory, "MySource");
+
+                Directory.CreateDirectory(mySourcePath);
+
+                // Generate Package
+                await SimpleTestPackageUtility.CreateFolderFeedV3Async(
+                    packageXPath,
+                    PackageSaveMode.Defaultv3,
+                    packageX);
+
+                string originalPackageXNupkg = Path.Combine(packageXPath, packageX.Id, packageX.Version, $"{packageX.Id}.{ packageX.Version}.nupkg");
+                string mySourcePackageXNupkg = Path.Combine(mySourcePath, $"{packageX.Id}.{ packageX.Version}.nupkg");
+
+                if (!File.Exists(originalPackageXNupkg))
+                {
+                    // On linux it's created in different place.
+                    string[] nupkgFiles = Directory.GetFiles(packageXPath, $"*.nupkg", SearchOption.AllDirectories);
+                    Assert.True(nupkgFiles.Count() == 1);
+                    originalPackageXNupkg = nupkgFiles[0];
+                }
+
+                // We're testing case we have only Nupkg file created at top.
+                File.Move(originalPackageXNupkg, mySourcePackageXNupkg);
+
+                // Remove packageXPath directory to prove we're not using nuget package structured folders.
```

Comment:
I'm not sure what's happening here. You create a package in a location, but then you deleted it and moved it to another one? 
Why the delete?

Reply:
See [above comment](https://github.com/NuGet/NuGet.Client/pull/3783#discussion_r536443187).
Point is just prove I'm not using v3 feed make this command working, instead using just nupkg files.
Because `dotnet pack --include-symbols --output ..\Mysource` command create just a nupkg file, nothing else.
So it's just work around to get nupkg file.

Reply:
This is no more concern by moving to V2 feed.